python/python_next/python_utils.h

```diff
@@ -0,0 +1,80 @@
+// clang-format off
+/*
+ * SPDX-FileCopyrightText: Copyright (c) 2025-present NVIDIA CORPORATION & AFFILIATES.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+// clang-format on
+#pragma once
+
+#include <exceptions.h>
+#include <cstddef>
+#include <cstdint>
+#include <optional>
+#include <utility>
+#include <vector>
+
+#include <runtime/executor_kernel_arg.h>
+#include <torch/csrc/utils/pybind.h>
+
+namespace nvfuser::python {
+
+// [ Note stride order and contiguity vector ]
```

Comment:
Duplicated utilities from `python_frontend`. I suppose I could make a common cpp folder between `python_frontend` and `python_next`. @jjsjann123

Reply:
I will move these utilities to nvfuser `csrc/`.

Reply:
I created `python/python_common/python_utils.h` to hold the helper functions shared between `python/python_direct` and `python/python_frontend`.